client/containers/Preference.jsx

```diff
@@ -111,28 +112,27 @@ class Preference extends Component {
 
   render () {
     return (
-      <div className="preference">
+      <div className="container">
```

Comment:
question: would this interfere with the line of code in client/index.js?

document.querySelector('.container')